### PR TITLE
fixes BUG 2275

### PR DESCRIPTION
**Pull request by BradReesWork**
Fixes the bug where "to_pandas_edgelist" ignores output column names

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

**Review by jnke2016 — APPROVED** [review]
Looks good to me

**Review by rlratzel — CHANGES_REQUESTED** [review]

File: python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py [hunk 1]
```diff
@@ -182,7 +182,8 @@ def __from_edgelist(
         if self.batch_enabled:
             self._replicate_edgelist()
 
-    def to_pandas_edgelist(self, source='source', destination='destination'):
+    def to_pandas_edgelist(self, source='src', destination='dst',
```
[the remark this answers is not in the document]
Comment (BradReesWork) [reply python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py:185]: true.  But the defaults really were not the default values used in the data structures.   
I'll update the doc string

**Comment by betochimas** [comment]
Observing from the current CI failures in `test_convert_matrix.py` and `test_multigraph.py`, some columns need to be renamed after this API change. In multiple tests from test_convert_matrix, the columns of res_pdf could be renamed:
`res_pdf = res_pdf.rename(columns={"source": "src", "destination": "dst"})`

**Review by jnke2016 — COMMENTED** [review]
It looks like this small change has some ramification on some tests

**Comment by BradReesWork** [comment]
the test in `test_convert_matrix` assume that the returned column names are "src", "dst", and "weight", so the updated function will help ensure that.  and I  fixed all the calls to DiGraph

**Review by rlratzel — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**